default.yml

```diff
@@ -203,11 +189,6 @@ Style/MethodDefParentheses:
 Lint/ShadowingOuterLocalVariable:
   Enabled: false
 
-# There are valid cases, for example debugging Cucumber steps,
-# also they'll fail CI anyway
-Lint/Debugger:
-  Enabled: false
```

Comment:
I think it is useful to have rubocop complain about this, even if CI will fail. The time I have these in my code _should_ be short lived, and stick out so I remember to remove them.

Reply:
I disagree on this one, I would prefer we lint and error even on `byebugs` and `pry` in our code.

Reply:
I think I misunderstood this cop; I _thought_ removing it meant it would flag it.

It turns out it must be set to `true`. As it is now, it is also not flagging it (at least in development, which I realized in the docs it shows as "okay")

My intent was to have it always flag so I'm going to set it to `true`.

Reply:
Oh it is changed to `true` but I also moved it alphabetically :P Didn't think about the repercussions for this discussion!

https://github.com/GetDutchie/Dutchie-Style/pull/4/files#diff-391be63d86ca0541cef3ee2c9302c75c391f294e4e8466af9c15d19137480fd1R40